micronaut-maven-integration-tests/src/it/dockerfile-docker-native-netty-custom-base/invoker.properties

```diff
@@ -0,0 +1,2 @@
+invoker.goals = -ntp mn:dockerfile -Dpackaging=docker-native
+invoker.os.family = !windows
```

Comment:
I know you saw this in other tests, but longer term the intention is to run all tests on Windows (see #809)

```suggestion
```